roles/kolibri/files/kolibri.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kolibri server
+
+[Service]
+User=ideascube
+Group=ideascube
+ExecStart=/home/ideascube/.kolibri/env/bin/python2 /home/ideascube/.kolibri/env/bin/kolibri start
```

Comment:
We (the Ideascube developers) have a strong preference for Python 3. In fact, Ideascube does not work any more with Python 2, and we have no intention of "fixing" that.

As a result, Python 3 must be installed on the boxes anyway for Ideascube, so moving everything to Python 3 means (potentially) less things installed on the boxes.

---

> From https://libraries.io/pypi/kolibri : both are acceptable

This seems like a third-party website, and as such might not be entirely accurate.

The best way to know what versions of Python are supported is to look at what is tested by the developers.

In the case of Kolibri, you can find this file in their source tree: https://github.com/learningequality/kolibri/blob/develop/tox.ini

As you can see, they run the tests on Python 2.7, 3.4 and 3.5. That's perfect for us, Debian Jessie has 3.4 and Stretch (if we ever move to it) has 3.5.